obspy/clients/nrl/client.py

```diff
@@ -0,0 +1,304 @@
+                                  subsequent_indent='    ')
+                info.extend(lines)
+        if self.dataloggers is None:
+            info.append('  Dataloggers not parsed yet.')
+        else:
+            info.append('  Dataloggers: {} manufacturers'.format(
+                len(self.dataloggers)))
+            if len(self.dataloggers):
+                keys = [key for key in sorted(self.dataloggers)]
+                lines = _textwrap("'" + "', '".join(keys) + "'",
+                                  initial_indent='    ',
+                                  subsequent_indent='    ')
+                info.extend(lines)
+        return '\n'.join(info)
+
+    def _repr_pretty_(self, p, cycle):
+        p.text(str(self))
+
+    def _choose(self, choice, path):
+        # Should return either a path or a resp
+        cp = self._get_cp_from_ini(path)
+        options = cp.options(choice)
+        if 'path' in options:
+            newpath = cp.get(choice, 'path')
+        elif 'resp' in options:
+            newpath = cp.get(choice, 'resp')
+        # Strip quotes of new path
+        newpath = self._clean_str(newpath)
+        path = os.path.dirname(path)
+        return self._join(path, newpath)
+
+    def _parse_ini(self, path):
+        nrl_dict = NRLDict(self)
+        cp = self._get_cp_from_ini(path)
+        for section in cp.sections():
+            options = sorted(cp.options(section))
+            if section.lower() == 'main':
+                if options not in (['question'], ['detail', 'question']):
+                    msg = "Unexpected structure of NRL file '{}'".format(path)
+                    raise NotImplementedError(msg)
+                nrl_dict._question = self._clean_str(cp.get(section,
+                                                            'question'))
+                continue
+            else:
+                if options == ['path']:
+                    nrl_dict[section] = NRLPath(self._choose(section, path))
+                    continue
+                # sometimes the description field is named 'description', but
+                # sometimes also 'descr'
+                elif options in (['description', 'resp'], ['descr', 'resp'],
+                                 ['resp']):
+                    if 'descr' in options:
+                        descr = cp.get(section, 'descr')
+                    elif 'description' in options:
+                        descr = cp.get(section, 'description')
+                    else:
+                        descr = '<no description>'
+                    descr = self._clean_str(descr)
+                    resp_path = self._choose(section, path)
+                    nrl_dict[section] = (descr, resp_path)
+                    continue
+                else:
+                    msg = "Unexpected structure of NRL file '{}'".format(path)
+                    raise NotImplementedError(msg)
+        return nrl_dict
+
+    def _clean_str(self, string):
```

Comment:
a bit overkill to have this function here, as far as I count it's only used 3 times